### PR TITLE
Regression: Avoid rendering unsupported media on PDFs & update quote styling

### PR DESCRIPTION
**Pull request by KevLehman**
## Proposed changes (including videos or screenshots)

- Fix faulty logic when deciding which attachment to render based on the mime type of it

- Replace media query syntax `(width < 767px)` with `(max-width: 767px)` since the first one isn't supported on safari
- Fix a bug when has bold text in multiple rows.

## Further comments

Add quotes style:
Screenshot 2023-02-16 at 22 31 58

Before:
Screenshot 2023-02-17 at 01 21 37
After:
Screenshot 2023-02-17 at 01 24 37

Since the library doesn't have support for inline styles and on the `Text` component margin, padding, and border won't work so we needed to simplify the inline code style component:
before:
Screenshot 2023-02-17 at 01 25 16

after:
Screenshot 2023-02-17 at 01 15 42

OC-674 OC-666 OC-646 OC-675 OC-694

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (murtaza98) [inline .vscode/launch.json:1]: Pushed this by mistake? ;-)

**Review by murtaza98 — APPROVED** [review]
LGTM!

**Review by aleksandernsilva — APPROVED** [review]

**Review by murtaza98 — DISMISSED** [review]

**Review by aleksandernsilva — APPROVED** [review]

File: apps/meteor/app/theme/client/imports/general/base_old.css [hunk 2]
```diff
@@ -1815,7 +1815,7 @@
 	}
 }
 
-@media (width <= 780px) {
+@media (max-width: 767px) {
```
Comment (ggazzo) [inline apps/meteor/app/theme/client/imports/general/base_old.css:1818]: I don't understand, why are these changes relevant?

File: apps/meteor/client/sidebar/RoomList/RoomList.tsx [hunk 3]
```diff
@@ -104,7 +104,7 @@ const RoomList = (): ReactElement => {
 			padding-block-start: 12px;
 		}
 
-		@media (width <= 400px) {
+		@media (max-width: 400px) {
```
Comment (ggazzo) [inline apps/meteor/client/sidebar/RoomList/RoomList.tsx:107]: same

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/app/theme/client/imports/general/base_old.css (hunk 2 above)
Comment (KevLehman) [reply apps/meteor/app/theme/client/imports/general/base_old.css:1818]: Not fully related to the PR title, they're a small fix for other regression.
Filipe can explain better, but AFAIK, Safari don't support the `w < x` instruction so some breakpoints were not being applied.

Comment (filipemarins) [reply apps/meteor/app/theme/client/imports/general/base_old.css:1818]: You are right Kevin. For this specific one, I also change the breakpoint value to match the fuselage value and fix some position issues.

**Review by murtaza98 — APPROVED** [review]

**Review by tassoevan — APPROVED** [review]

**Pull request merged**